onnxruntime/core/mlas/lib/quantize.cpp

```diff
@@ -422,3 +422,37 @@ Return Value:
 }
 
 #endif
+
```

Comment:
can you make this look more MlasComputeTanh? The comment style is not the same, ) is not done the same,